Policies/github-subprocessors-and-cookies.md

```diff
@@ -0,0 +1,110 @@
+| `:tz` | This cookie allows your browser to tell us what time zone you're in. |
+| `:gist_user_session` | This cookie is used by Gist when running on a separate host. |
+| `:saml_csrf_token` | This cookie is set by SAML auth path method to associate a token with the client. |
+| `:saml_return_to` | This cookie is set by the SAML auth path method to maintain state during the SAML authentication loop. |
+| `:gist_oauth_csrf` | This cookie is set by Gist to ensure the user that started the oauth flow is the same user that completes it. |
+| `:"__Host-user_session_same_site"` | This cookie is set to ensure that browsers that support SameSite cookies can check to see if a request originates from GitHub. |
+| `:"__Host-gist_user_session_same_site"` | This cookie is set to ensure that browsers that support SameSite cookies can check to see if a request originates from GitHub. |
+| `:_ga` | This cookie is used by Google Analytics. |
+| `:_octo` | This cookie is used by Octolytics, our internal analytics service, to distinguish unique users and clients. |
+| `:tracker` | This cookie tracks the referring source for signup analytics. |
+
+Certain pages on our site may set other third party cookies. For example, we may embed content, such as videos, from another site that sets a cookie. While we try to minimize these third party cookies, we can’t always control what cookies this third party content sets.
+
+### Tracking on GitHub
+
+"[Do Not Track](https://www.eff.org/issues/do-not-track)" is a privacy preference you can set in your browser if you do not want online services — specifically ad networks — to collect and share certain kinds of information about your online activity from third party tracking services. GitHub does not currently respond differently to an individual browser's Do Not Track setting. If you would like to set your browser to signal that you would not like to be tracked, please check your browser's documentation for how to enable that signal. There are also good applications that block online tracking, such as [Privacy Badger](https://www.eff.org/privacybadger). 
+
+We do not track your online browsing activity on other online services over time and we do not host third-party advertising on GitHub that might track your activity on our site. We do have agreements with certain vendors, such as analytics providers, who help us track visitors' movements on certain pages on our site. Only our vendors, who are collecting data on our behalf, may collect data on our pages, and we have signed data protection agreements with every vendor who collects this data on our behalf. We use the data we receive from these vendors to better understand our visitors' interests, to understand our website's performance, and to improve our content. Any analytics vendor will be listed in our Subprocessor List above, and you may see a list of every page where we collect this kind of data below.
```

Comment:
It'd be nice to have a list of all tracking scripts on GitHub in μBlock Origin - compatible format ;)

Reply:
That should be pretty easy to build, though, with the information provided here.

Reply:
I prefer GitHub providing a one as a gesture of goodwill :) Of course it doesn't prevent us from having an own one.

Reply:
We'll give it some thought, but we probably won't show partiality to one block format over another, and y'know, we're an open source community, we do like to encourage some independence and sharing. 😁